### ISSUE TITLE
Postgresql to Clickhouse 

### ISSUE THREAD

**Issue by meysammeisam** [Altinity/clickhouse-sink-connector#597]
This project supports toolsets to Dump Mysql data and then restore it to Clickhouse.
We are using it for our Mysql replications, but I'm wondering what would be the solution for the Postgresql DBs.
Does anyone know any workaround or toolset to cover that?
So the idea is to:

- Start the Sink Connector Light in schema-only mode to incrementally replicate the data from the source DB to CH
- Dump the rest of the data and restore it to Clickhouse manually

As mentioned before, we do the same with Mysql DBs and now looking for a similar solution for our Postgres DBs.

**Comment by aadant** [comment]
@meysammeisam indeed, it is a cool feature. We would also need to implement the checksum for Postgres and the fast table counts.

This project is open source, feel free to contribute !

**Comment by minguyen9988** [comment]
Implemented at develop head `e123450c8`. The PostgreSQL equivalents of the MySQL dump/load and comparison tooling now exist:

- `sink-connector/python/ch_sink_tools/db_dump/postgres_dumper.py` — parallel PostgreSQL to ClickHouse snapshot loader; it captures the pre-snapshot WAL LSN and writes it to the offset table so CDC resumes at the right position (`def write_lsn_offset(ch_conn, offset_table, lsn_int, connector_name, dry_run=False):`).
- `sink-connector/python/ch_sink_tools/db_compare/postgres_table_checksum.py:6` — "compute a deterministic checksum for a PostgreSQL table".
- `sink-connector/python/ch_sink_tools/db_compare/postgres_table_count.py:6` — "compute exact COUNT(*) for a PostgreSQL table", covering the fast table counts asked for in the comment thread.

Console entry points are registered in `sink-connector/python/pyproject.toml:38-44` (`ch-pg-checksum`, `ch-pg-count`, `ch-pg-dump`). Added in commit `3a79f1b2a`, confirmed an ancestor of develop.
